packages/esm-patient-list-management-app/src/add-patient/add-patient.component.tsx

```diff
@@ -166,12 +169,13 @@ const AddPatient: React.FC<AddPatientProps> = ({ closeModal, patientUuid }) => {
       {paginated && (
         <div className={styles.paginationContainer}>
           <span className={classNames(styles.itemsCountDisplay, styles.bodyLong01)}>
-            {results.length * currentPage} / {searchResults.length} {t('items', 'items')}
+            {searchResults.length > 0 ? `${startIndex}-${endIndex} / ${searchResults.length}` : '0'}{' '}
+            {t('items', 'items')}
           </span>
           <Pagination
             className={styles.pagination}
-            forwardText=""
-            backwardText=""
+            forwardText={t('nextPage', 'Next page')}
+            backwardText={t('previousPage', 'Previous page')}
```

Comment:
I think we should keep these labels blank ("") because the tooltips currently get cut off by the modal container. 

![CleanShot 2025-03-06 at 11  56 41@2x](https://github.com/user-attachments/assets/bc694112-b7e1-4d21-80aa-d97532286725)

The pagination API doesn't provide control over tooltip positioning/alignment, so leaving them blank is the safer option until we have a better solution for the overflow issue.